sdk/python/setup.py

```diff
@@ -32,7 +32,7 @@
     'google-auth>=1.6.1',
     'requests_toolbelt>=0.8.0',
     'cloudpickle==1.1.1',
-    'kfp-server-api >= 0.1.18, <= 0.1.40',  #Update the upper version whenever a new version of the kfp-server-api package is released. Update the lower version when there is a breaking change in kfp-server-api.
+    'kfp-server-api >= 0.2.5, <= 0.2.5',  #Update the upper version whenever a new version of the kfp-server-api package is released. Update the lower version when there is a breaking change in kfp-server-api, or kfp sdk depends on new api changes in kfp-server-api.
```

Comment:
would 'kfp-server-api >= 0.1.18, <= 0.2.5' work? I don't know the context either I guess it's for backward compatibility purpose. a new sdk can talk to old version of KFP server.
and i assume this would be a safer change.

Reply:
https://github.com/kubeflow/pipelines/pull/3201/files#diff-1f65a44a2187458da0f00e0ebbe730e5R470
If using new kfp sdk with old kfp-server-api, `schedule_pipeline` method call will always call because it uses the new field, but old method doesn't have it.

I think this can be considered as breaking change, @numerology for double confirming this.

What do you think?

Reply:
Breaking change LGTM.

Can we make sure to highlight this in the release note? Thanks

Reply:
SGTM, I will highlight this